test/test_slave.html

```diff
@@ -31,11 +32,13 @@
     <div id="end"></div>
   </body>
   <script>
-    require.config({paths: {'pdfjs': '../src'}});
-    require(['pdfjs/display/api', 'pdfjs/display/text_layer',
-             'pdfjs/display/annotation_layer', 'pdfjs/display/global',
-             'pdfjs/shared/util'],
-        function (api, textLayer, annotationLayer, global, pdfjsSharedUtil) {
+    Promise.all([SystemJS.import('pdfjs/display/api'),
+                 SystemJS.import('pdfjs/display/text_layer'),
+                 SystemJS.import('pdfjs/display/annotation_layer'),
+                 SystemJS.import('pdfjs/display/global'),
+                 SystemJS.import('pdfjs/shared/util')])
+          .then(function (modules) {
+      var pdfjsSharedUtil = modules[4];
```

Comment:
I think we can get rid of this intermediate variable and make the next line `window.pdfjsSharedUtil = modules[4];` directly. The intermediate variable doesn't provide more clarity about what's happening.